### PR TITLE
Themes color and variables

### PR DESCRIPTION
**Pull request by sonAndrew**
Hey, @samith-s , I only had time to fix :

1. **theme variables** 
2. **dark theme colors**

Inside the **_src/themes_** folder : 

> I created a folder for **partials** 
> Inside partials a folder for **variables**

Inside **_/themes/partials/variables_** folder :

> I created files for each dark and light variables

screenshot 2018-08-23 01 02 06

Inside the **_themes/dark.scss_** file : 
> Imported /partials/variables/_dark-variables

screenshot 2018-08-23 01 03 02

Inside the **_themes/light.scss_** file : 
> Imported /partials/variables/_light-variables

screenshot 2018-08-23 01 02 20

### CODE REVIEW

**Comment by samrith-s** [comment]
@sonAndrew Hey, there is a build error. Please check it out once. Will review once the build succeeds. I think the import path for dark-variables is incorrect.

**Comment by sonAndrew** [comment]
Hey, @samrith-s I checked to see if the dark theme worked but because of the fix to the variable names and colors, it gives a weird offset that doesn't look like the file attached on GitHub
screenshot 2018-08-23 23 45 20
.

**Comment by sonAndrew** [comment]
Hey, @samrith-s I checked to see if the dark theme worked but because of the fix to the variable names and colors, it gives a weird offset that doesn't look like the file attached on GitHub. If you can tell me what you want each to be I will fix it. I have a color palette on codepen. Click [HERE](https://codepen.io/Atsui-yo/pen/mGJpqq) !

File: src/themes/partials/variables/_dark-variables.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+$dark: #222222;
```
Comment (samrith-s) [inline src/themes/partials/variables/_dark-variables.scss:1]: Do a check of the variables. The dark theme looks like a light theme with a mix of dark backgrounds. Please verify this.

File: src/themes/partials/variables/_light-variables.scss [hunk 2]
```diff
@@ -0,0 +1,7 @@
+$dark: #333;
+$interaction: #add8e6;
```
Comment (samrith-s) [inline src/themes/partials/variables/_light-variables.scss:2]: This either has to be `interaction-color` or all dependencies in the light theme stylesheet have to be changed to `interaction`. It throws an error when you use the light theme.

Please check it thoroughly for both themes.

**Review by samrith-s — CHANGES_REQUESTED** [review]
@sonAndrew I'm a bit confused by this. What exactly is the *offset* you are talking about?

Also, please remove `package-lock.json`. We are using `yarn` for this project not `npm`.

**Review by samrith-s — APPROVED** [review]
Looks good!

**Pull request merged**